DuckDuckGo/MainViewController+KeyCommands.swift

```diff
@@ -33,7 +33,7 @@ extension MainViewController {
         }
         
         var browsingCommands = [UIKeyCommand]()
-        if currentTab != nil {
+        if homeController == nil {
```

Comment:
This is very good change - I think I recall seeing some crashes related to the `currentTab` call having a side effects of creating tab instance.